gazebo_plugins/include/gazebo_plugins/gazebo_ros_ft_sensor.h

```diff
@@ -88,7 +90,7 @@ class GazeboRosFT : public ModelPlugin
 
   /// \brief Gaussian noise
   private: double gaussian_noise_;
-  private: unsigned int seed;
+  private: std::default_random_engine seed;
```

Comment:
These changes break ABI. Do you think you could make this patch ABI-compatible?

Reply:
Let me see what I can do here.

Reply:
after speaking with @chapulina we think that although breaking ABI is something that we try to avoid, the ROS policy allows it and we should not stop changes specially if not breaking ABI involves too many effort. We thinks is fine to break it in this case. Thanks anyway Sean.